### DISCUSSION TITLE
Build Images Are Quite Large—Is This Normal?

### DISCUSSION THREAD

**Discussion by rayblair06** [php/frankenphp#1123] [Q&A] [answered]
Hey, everyone!

First off, thanks for all your hard work on FrankenPHP. Overall, it’s been a great experience! I’ve been testing FrankenPHP on a small project, and I noticed the Docker images are fairly large, around `2.9GB`, with the project being only `38MB`. I understand that FrankenPHP includes Caddy and other dependencies, but is this size typical, or have I misconfigured something?

Here’s my Dockerfile for reference: https://github.com/rayblair06/dumbo-qr/blob/main/Dockerfile

**Comment by dunglas** [comment, marked as answer]
It's because you're using the builder image. This image includes a large number of build dependencies.

Use the runner image (which is small) in production (remove the `builder-` part of the tag).

**Reply by rayblair06** [reply]
My mistake, thank you for that!
